lizmap/modules/lizmap/lib/App/JelixContext.php

```diff
@@ -255,4 +267,12 @@ public function getFullUrl($selector, $params = array())
     {
         return \jUrl::getFull($selector, $params);
     }
+
+    public function getIniModifier($ini)
+    {
```

Comment:
I think the real content of this method is missing. there is no result property, no use of jIniModifier etc...